README.md

```diff
@@ -9,14 +9,19 @@ Image processing and manipulation in JavaScript.
 
 ## Installation
 
-`$ npm i image-js`
+`$ npm i image-js@next`
```

Comment:
```suggestion
`$ npm i --save-exact image-js@next`
```

The prerelease versions are based on commit hash and do not follow incremental numbers